chapter_multilayer-perceptrons/mlp-scratch_vn.md

```diff
@@ -46,15 +46,21 @@ Note that we can regard both of these quantities as *hyperparameters* and ought
 Typically, we choose layer widths in powers of $2$ which tends to be computationally efficient because of how memory is alotted and addressed in hardware.
 -->
 
-*dịch đoạn phía trên*
+Nhắc lại rằng Fashion-MNIST gồm có $10$ nhãn, mỗi ảnh gồm có một lưới $28 \times 28 = 784$ điểm ảnh (đen và trắng).
+Chúng ta sẽ tiếp tục bỏ qua liên hệ về mặt không gian giữa các điểm ảnh (ở thời điểm hiện tại), khi đó ta có thể coi nó đơn giản như một tập dữ liệu phân loại với $784$ đặc trưng đầu vào và $10$ lớp.
```

Comment:
```suggestion
Chúng ta sẽ tiếp tục bỏ qua liên hệ về mặt không gian giữa các điểm ảnh (ở thời điểm hiện tại), khi đó ta có thể xem nó đơn giản như một tập dữ liệu phân loại với $784$ đặc trưng đầu vào và $10$ lớp.
```

Reply:
`coi` là `consider`, `xem` là `see` :)

Reply:
@tiepvupsu @aivivn Anh có thể nói kỹ hơn ý của anh trong câu chuyện này không ạ? Em nghĩ em chưa rõ được ý của anh ạ.

Hai từ này trong tiếng Việt là từ đồng nghĩa với ngữ nghĩa trong phần lớn ngữ cảnh là tương đồng nhau. Và hai từ này còn là từ đơn nữa, nên `consider` nếu dịch là `coi` không thì không hề đủ nghĩa; khi đặt trong ngữ cảnh kỹ thuật nếu tìm một từ mang nghĩa `consider` thì có thể tìm tới từ cân nhắc, suy nghĩ kỹ, chú ý đến hoặc là xem xét... nó là từ ghép.

Với từ `coi` em thấy từ mặc dù đồng nghĩa nhưng giới hạn sử dụng của nó lại không rộng bằng `xem`, và cũng vì nó là từ đơn, đồng nghĩa nên em mới có ý chọn từ để thống nhất hơn khi lựa chọn sử dụng.

Reply:
Thực ra ngày đi học anh hay dùng từ `coi` khi làm toán. Ít khi dùng từ `xem` mặc dù anh là người Bắc.

Từ `consider` vừa có nghĩa là `xem xét` vừa có thể hiểu là `coi như`. Trong trường hợp này nó mang nghĩa thứ hai. 

Anh vote giữ nguyên từ `coi`. Dù sao thì từ này ở đây cũng không quan trọng lắm.